tests/test_operator.py

```diff
@@ -1074,3 +1074,77 @@ def test_explicit_run(self):
         args['a'] = array
         op.cfunction(*list(args.values()))
         assert all(np.allclose(args['a'][i], i) for i in range(time_dim))
+
+
+@skipif_yask
+class TestLoopOrder(object):
```

Comment:
Tests for loop iteration direction should go in ``class TestLoopScheduler``

Tests for correctness when iterating over sub-regions should go in ``class TestRegions``